README.md

```diff
@@ -1,114 +1,45 @@
-# [EN] Grav for YunoHost
-
+[![Integration level](https://dash.yunohost.org/integration/grav.svg)](https://ci-apps.yunohost.org/jenkins/job/grav%20%28Community%29/lastBuild/consoleFull)
 [![Install Grav with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=grav)
 
-* http://getgrav.org/
-* https://github.com/getgrav/grav
-
-## Installation
-
-From YunoHost admin panel:
-
-1. Enter repository url https://github.com/YunoHost-Apps/grav_ynh
-2. Configure the app
-3. Click install
-4. Go to the url you configured
-
-From command line:
-
-```sh
-sudo yunohost app install -l grav https://github.com/YunoHost-Apps/grav_ynh
-```
-
-Options:
-
-1. __Choose a domain for Grav__  
-Domain for your installation (example.org).
-
-2. __Choose a path for Grav__  
-Path for your installation (/grav).
-
-3. __Is it a public application ?__  
-If public, everybody will be able to access your Grav instance (example.org/grav).
-
-4. __Do you wish to install the Admin plugin ?__  
-Grav has an optional admin panel, answer `Yes` if you want to use it.
-
-5. __Make admin panel private ?__  
-_Only valid if admin panel is installed._  
-Protect the admin panel behind Yunohost authentication.
-
-6. __Create administrator automatically ?__  
-_Only works if admin panel is installed and private._  
-If `Yes`, an admin account will be automatically created when you first visit your Grav instance, and you will be redirected to the admin panel. Your authentication will be handled by your YunoHost connection.  
-If `No`, on your first visit to your Grav instance Grav will propose you to create a new user.
-
-<hr/>
+Grav is a modern open source flat-file CMS.
 
-# [FR] Grav pour YunoHost
+> *This package allow you to install Grav quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.*
 
 * http://getgrav.org/
 * https://github.com/getgrav/grav
 
-## Installation
-
-Depuis le panneau d'administration YunoHost:
+Provided version: **1.5.1**
 
-1. Entrez l'url du dépot https://github.com/YunoHost-Apps/grav_ynh
-2. Configurez l'application
-3. Cliquez sur installer
-4. Rendez-vous à l'url que vous avez configuré
+## Installation
 
-Depuis la ligne de commande:
+Use either YunoHost's admin panel or command line:
 
 ```sh
-sudo yunohost app install -l grav https://github.com/YunoHost-Apps/grav_ynh
+sudo yunohost app install --url grav https://github.com/YunoHost-Apps/grav_ynh
```

Comment:
`--url` is not listed when I do `yunohost app install -h`. I suggest the following as a minimal command:
```
sudo yunohost app install https://github.com/YunoHost-Apps/grav_ynh
```